packages/create-skill-app/create-app.ts

```diff
@@ -90,9 +90,22 @@ export async function createApp({
     },
   })
 
+  const defaultDevPort = Number(3000).toString()
+
   const packageJson = {
     name: appName,
     ...defaultPackage,
+    scripts: {
+      ...defaultPackage.scripts,
+      dev: defaultPackage.scripts.dev.replace(
+        defaultDevPort,
+        projectData.devPort,
+      ),
+      start: defaultPackage.scripts.start.replace(
+        defaultDevPort,
+        projectData.devPort,
+      ),
+    },
```

Comment:
so I ended up processing it like this instead. `dev` and `start` commands will both get the correct dev port.